woocommerce/woocommerce-ios
# Order list: feature flag and top bar that display order filters

Part of #5243

## Description
As part of the Order List filters project, I added a new feature flag that will be useful to hide/show the new top bar in the Order List that will be useful for displaying and editing the filters on the order list.

## Testing
1. Open the app, and press the "Orders" tab.
2. You should see the new top bar that displays the label "All Orders" on the left, and a button "Filter" on the right.
3. Go into the code, and edit the feature flag `orderListFilters` under `DefaultFeatureFlagService`, changing the return to `false`.
4. Repeat from the beginning. This time, the top bar should be hidden.

## Screenshots
| Light mode             |  Dark mode |
:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 13 Pro - 2021-10-20 at 15 57 59](https://user-images.githubusercontent.com/495617/138108109-afbb6f51-afdc-4a15-9f75-768bbd99c3c8.png) | ![Simulator Screen Shot - iPhone 13 Pro - 2021-10-20 at 15 57 50](https://user-images.githubusercontent.com/495617/138108120-edccff40-674a-4056-a2d2-396852fbd2a0.png)

Update release notes:
